docs/root-manifest.json

```diff
@@ -83,6 +83,30 @@
 		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/extensibility/autocomplete.md",
 		"parent": "extensibility"
 	},
+	{
+		"title": "Design",
+		"slug": "design",
+		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/design.md",
+		"parent": null
+	},
+	{
+		"title": "Patterns",
+		"slug": "patterns",
```

Comment:
Similar to `resources` - patterns is very short for the slug name, I think we should we do the following:

File renamed to `design-patterns.md`.

```json
"slug": "design-patterns",
"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/design\/design-patterns.md",
```